contracts/StorageManager.sol

```diff
@@ -1,12 +1,15 @@
 pragma solidity 0.6.2;
+pragma experimental ABIEncoderV2;
 
 import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
Why?

Reply:
AFAIK when deploying through Proxy you need the versions that have the `initialize` function instead of constructors.  The package they recommend to be used for Upgradable contracts is: https://github.com/OpenZeppelin/openzeppelin-contracts-ethereum-package. "  
There is an explanation here: https://docs.openzeppelin.com/upgrades/2.8/writing-upgradeable and they have an example with ERC20, but I guess in our case we are not instantiating or deploying those so maybe its not required. This seems required if you plan to get the constructors of the contracts executed or if you expect to upgrade those contracts.

Reply:
Ah yes. No, in this case it is indeed not needed, as we are only using the ERC20 for the interface (so our contract knows how to interact with it). Besides, this PR does not add upgradability.